study1/week2/LongestPalindrome.java

```diff
@@ -0,0 +1,28 @@
+class LongestPalindrome {
+
+    public int longestPalindrome(String s) {
+        int n = s.length();
+        int odd = 0;
```

Comment:
Per the [style guide](https://google.github.io/styleguide/javaguide.html#s4.8.2-variable-declarations) please move this `odd` variable to line 13.